src/main/java/haveric/recipeManager/recipes/CraftRecipeParser.java

```diff
@@ -88,7 +88,7 @@ public boolean parseRecipe(int directiveLine) throws Exception {
 
         recipe.setIngredients(ingredients); // done with ingredients, set 'em
 
-        if (recipe.hasFlag(FlagType.REMOVE)) {
+        if (recipe.hasFlag(FlagType.REMOVE) && !Version.has1_12Support()) {

```

Comment:
Yeah, I'm not sure. It shouldn't take too long to fix on the user end (unless they are removing most recipes) so might not be worth a deeper dive